libs/testdata/src/main/kotlin/grunnlag/GrunnlagTestData.kt

```diff
@@ -39,6 +40,36 @@ data class GrunnlagTestData(
                         AvdoedesBarn(listOf(soeker, soesken, halvsoesken)).toJsonNode(),
                     ),
             )
+
+    private val avdoedesBarnMedEnDoed
```

Comment:
Dette er ekstremt rigid å jobbe med, skjønner ikke helt hvorfor vi ikke kan bruke person objektet direkte i stedet for via konstantopplysninger og annen magisk wrapping i familie og hele grunnlagsobjektet. Er vel en grunn til at grunnlag ikke har noen ordentlige tester.